CLOUDPROVIDERS.md

```diff
@@ -0,0 +1,98 @@
+Configuring cloud providers
+--------------------
+Cloud providers are a way to automatically enrich metrics with metadata from a cloud vendor.
+
+Cloud providers must be configured through the usage of a configuration file (toml, yaml and json are supported), passed via
+`--config-path`.
+
+There are currently two supported cloud providers:
+
+* `aws` which retrieves tags from AWS instance tags via AWS API calls.
+* `k8s` which retrieves tags from pod labels and annotations.
+
+All configuration is in a stanza named after the backend, and takes simple key value pairs.
+
+**Cloud providers should be disabled on the aggregation server when using http forwarding, as the source IP isn't
+propagated, and that information should be collected on the ingestion server.**
+
+aws
+---
+### TODO
+
+k8s
```

Comment:
We should include a line about the host behavior and pod name, and the implications of it.  It's called out in the examples, but there's no context there for it.